task9/index.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import {config} from "./config/config.js";
+import meteorRouter from './delivery/meteorController.js'
+import errorHandler from './exception/errorHandler.js';
+
+const app = express();
+
+app.use(meteorRouter, errorHandler)
```

Comment:
you forgot to remove errorHandler from `app.use(meteorRouter, errorHandler)` :)

Reply:
oops, fixed